xonsh/environ.py

```diff
@@ -1205,7 +1205,11 @@ def DEFAULT_DOCS():
             "* Stopping the thread with ``Ctrl+Z`` yields to job control.\n"
             "* Threadable commands are run with ``Popen`` and threadable \n"
             "  alias are run with ``ProcProxy``.\n\n"
-            "The desired effect is often up to the command, user, or use case."
+            "The desired effect is often up to the command, user, or use case.\n\n"
+            ".. note: This is ``False`` by default on Windows as as this feature has\n"
```

Comment:
Here's a typo in "as as".

Reply:
Thanks. I will fix it :)

Reply:
Now there's a double space instead of double "as", but I don't think it really matters ;)